beanie/odm/interfaces/inheritance.py

```diff
@@ -0,0 +1,31 @@
+from typing import ClassVar
+from pydantic import BaseModel
+
+
+class InheritanceInterface:
```

Comment:
Yes, it is only used once in the Document class. I just follow similar approaches to make the code look like a single style. So it can be injected inside the Document class.